### DISCUSSION TITLE
Stretching animation length

### DISCUSSION THREAD

**Discussion by MrKacafirekCZ** [MrKacafirekCZ/doggo-mod-overhauled#40] [Polls]
A suggestion (#39) mentioned that some animations are too long, particularly the stretching animation. I would like to know if it is actually too long. And as a bonus question whether or not it happens way too often.

**Comment by Some-Doofus** [comment]
Just tried the mod out for the first time and came here to say exactly this. I like the animations a lot, but I feel they are far too long and happen too often. The stretch and scratch animations should only last about 2-5 seconds and should only play once every couple minutes imo. Dogs don't typically scratch and/or stretch so often or for so long irl, but the more minor animations like sniffing and head turning can be more frequent (ie the way they are now is fine).

**Reply by MrKacafirekCZ** [reply]
I can definitely make scratching and stretching happen less often however decreasing the length is much more difficult than it looks. The length actually depends on the animation itself and animations are just whole bunch of math function put together. Just one single animation can take multiple days of work just to get it right, it's very time consuming.
